about.html

```diff
@@ -0,0 +1,150 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css" />
+    <title>Moroccan Kitchen</title>
+</head>
+<body>
+    <header>
+        <div class="social-media">
+          <ul class="social-icons">
+            <li class="icon">
+              <a href="#">
+                <img src="./Assets/insta.svg" alt="Instagram Icon" />
+              </a>
+            </li>
+            <li class="icon">
+              <a href="#"
+                ><img src="./Assets/twitter.svg" alt="Twitter Icon"
+              /></a>
+            </li>
+            <li class="icon">
+              <a href="#">English</a>
+            </li>
+            <li class="icon">
+              <a href="#">My Page</a>
+            </li>
+            <li class="icon">
+              <a href="#">Logout</a>
+            </li>
+          </ul>
+        </div>
+  
+        <nav class="main-header">
+          <img class="hamburger" src="./Assets/menu.svg" alt="Menu Icon" />
+          <a class="logo-container" href="#">
+            <img src="./Assets/logo.svg" alt="Logo Icon" />
+            <h3>The Moroccan Kitchen.</h3>
+          </a>
```

Comment:
Let's try to make sure the page logo leads us to the homepage every time as this can lead to a better user experience. kindly add the link to the index page on the `href` 👍